src/gen_dispatch.py

```diff
@@ -816,14 +816,22 @@ def write_source(self, f):
             self.write_thunks(func)
         self.outln('')
 
+        self.outln('#define EPOXY_DISPATCH_PTR(name) name##_global_rewrite_ptr')
+
         self.outln('#if USING_DISPATCH_TABLE')
 
+        self.outln('#undef EPOXY_DISPATCH_PTR')
+        self.outln('#define EPOXY_DISPATCH_PTR(name) name##_dispatch_table_thunk')
+
         self.outln('static struct dispatch_table resolver_table = {')
         for func in self.sorted_functions:
             self.outln('    epoxy_{0}_dispatch_table_rewrite_ptr, /* {0} */'.format(func.wrapped_name))
         self.outln('};')
         self.outln('')
+        """
```

Comment:
You've added `EPOXY_STATIC_BUILD`, but now you've commented out the whole block using it.

Reply:
The new flag is now removed since `__thread` also applies to shared library.